src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/Gauge.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Diagnostics.Metrics
+{
+    /// <summary>
+    /// The Gauge is an instrument used to record non-additive values whenever changes occur. For example, record each time the number of active threads changes.
```

Comment:
I think the OTel spec uses temperature measurement as an example for Gauge and I think that would be better suited here.

The number of active threads is sometimes useful to handle additively - for example its reasonable to say "My fleet of computers uses a total of 745 threads to serve all my web requests." Temperature doesn't make sense to sum up - as in "My fleet of computers has CPU temperatures which sum to 3,821 degrees"

Reply:
Thanks, I used the example used in the OTel specs. `record the room background noise level value when changes occur.`. Thanks!

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#gauge